packages/paste-core/components/menu/src/MenuItem.tsx

```diff
@@ -17,7 +17,13 @@ export const StyledMenuItem = React.forwardRef<HTMLDivElement | HTMLAnchorElemen
         appearance="none"
         background="none"
         border="none"
-        color={props['aria-disabled'] ? 'colorTextWeaker' : 'colorText'}
+        color={
+          props['aria-disabled']
```

Comment:
question: is there a reason we'd use `props['aria-disabled']` to detect if the menu item is disabled? I copied this from the previous ternary operator, but tried `props.disabled` and that seemed to work similarly

Reply:
i would add the disabled styles to the _disabled selector below on line 47, that includes both disabled and aria-disabled